src/plugins/intel_gna/orientation_helper.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <legacy/ie_layers.h>
+
+#include <ie/ie_input_info.hpp>
+#include <string>
+
+#include "backend/dnn_components.hpp"
+#include "descriptions/gna_desc.hpp"
+
+namespace GNAPluginNS {
+/**
+ * @namespace helpers contains helpers tools for gna plugin.
```

Comment:
Didn't get do you mean to use namespace:
intel::gna_plugin::orientation_helper  instead of intel::gna_plugin::helpers?
I think this name space could be extended or removed in the future. Depends if we decided to refactor plugin?

Reply:
> Didn't get do you mean to use namespace: intel::gna_plugin::orientation_helper instead of intel::gna_plugin::helpers? I think this name space could be extended or removed in the future. Depends if we decided to refactor plugin?

This comment about file names. If this files intended to contain orientation related functions only it is ok.

Reply:
can we reevaluate this comment?